CMakeLists.txt

```diff
@@ -223,17 +223,17 @@ add_subdirectory(docs)
 
 # DESTINATION gets the previous path and appends the argument --> /opt/local/GrPPI
 
-# cmake -DCMAKE_INSTALL_PREFIX=path/to/folder
+# cmake .. -DCMAKE_INSTALL_PREFIX=path/to/folder
 
 #llvm --> cmake -DCMAKE_INSTALL_PREFIX=/tmp/llvm 
 
 # To compile once installed
 # g++ farm1.cpp -I /tmp/GrPPI -std=c++14 -lboost_system -lboost_thread
 
 # Installation -- make install
-install ( DIRECTORY ${PROJECT_SOURCE_DIR}/include DESTINATION  "${CMAKE_PROJECT_NAME}" )
+install ( DIRECTORY ${PROJECT_SOURCE_DIR}/include/ DESTINATION  "include/${CMAKE_PROJECT_NAME}" )
 # Installation of fastflow
-install ( DIRECTORY ${PROJECT_SOURCE_DIR}/fastflow DESTINATION "${CMAKE_PROJECT_NAME}" )
+# install ( DIRECTORY ${PROJECT_SOURCE_DIR}/fastflow DESTINATION "${CMAKE_PROJECT_NAME}" )
 
 #install( FILES ${PPIHEADERS_CPU} DESTINATION /include/include/ )
 #install( FILES ${PPIHEADERS_GPU} DESTINATION /include/include/ )
```

Comment:
That was the way of making the install at the beginning of the CMake, there are two lists with the headers. If we want to have an installation of certain files it could be useful, otherwise we can remove it.